### DISCUSSION TITLE
Difference in animations of 300s/100s/50s in skin from stable

### DISCUSSION THREAD

**Discussion by asdsadv** [ppy/osu#20723] [Q&A] [answered]
I'm using the main ekoro skin which is linked in this doc. https://docs.google.com/document/d/1g24iPeOJX4VAO4zmL9JiQgfjRUjIm6iM5kT9uwiW2Nw

The animations on the 300s/100s/50s is significantly different in lazer compared to standard. I've taken a screenshot on both using auto, in approximately the same part of a map.
Standard:
![Screenshot (740)](https://user-images.githubusercontent.com/62586440/195280035-c79c2fe9-2bf3-4a14-bc6e-1c7d4f092cd6.png)
Lazer:
![Screenshot (741)](https://user-images.githubusercontent.com/62586440/195280056-1b585981-1c41-424d-b04c-ce1f1fa273b5.png)

**Comment by peppy** [comment, marked as answer]
This is because lazer is using the high resolution sprites from the skin. You'll find that stable looks the same if you increase the window resolution.

Note the different in definition between the two files:

![particle300](https://user-images.githubusercontent.com/191335/195297360-ad39c4ce-9040-45a2-9969-bf6b9f3b47a6.png)
![particle300@2x](https://user-images.githubusercontent.com/191335/195297369-7429f112-a1ae-4e1f-8f01-3e7852410771.png)


If you'd like the old look back, take your `particle.png` and upscale it to `particle@2x.png`

**Reply by asdsadv** [reply]
Thank you for the answer. In this case it seems like most of the difference was just me being dumb, and having hit lighting on in lazer while I had it off in stable.
